.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,8 @@
+- id: sort
+  name: Check Cargo.toml is sorted
+  description: 'Ensure Cargo.toml is sorted'
+  entry: cargo-sort -c -w
```

Comment:
Can this just be `cargo-sort` and when someone adds a pre-commit hook they can add `args: ["-cw"]` or whatever they need?

Reply:
According to the pre-commit docs that seems fine. The command should either fail or change some files so that would work without any args.

I though of `-c` and `-w` being good default arguments, although it's maybe better to put them in the `args` entry so they can be overwritten by the user.

Running without any args is probably less confusing as the pre-commit plugin would behave exactly like using the CLI tool directly. Therefore, I'll remove the args 👍